astropy/coordinates/earth.py

```diff
@@ -235,6 +270,73 @@ def of_site(cls, site_name):
             return newel
 
     @classmethod
+    def from_address(cls, address, get_height=False):
+        """
+        Return an object of this class for a given address by querying the Google
+        maps geocoding API.
+
+        This is intended as a quick convenience function to get fast access to
+        locations. In the background, this just issues a query to the Google maps
+        geocoding API. It is not meant to be abused! Google uses IP-based query
+        limiting and will ban your IP if you send more than a few thousand queries
+        per hour. See the Google maps geocoding documentation for more information:
+        https://developers.google.com/maps/documentation/geocoding/intro
+
+        .. warning::
+            If the query returns more than one location (e.g., searching on
+            ``address='springfield'``), this function will use the _first_ returned
+            location.
+
+        Parameters
+        ----------
+        address : str
+            The address to get the location for. As per the Google maps API, this
+            can be a fully specified street address (e.g., 123 Main St., New York,
+            NY) or a city name (e.g., Danbury, CT), or etc.
+        get_height : bool (optional)
+            Use the retrieved location to perform a second query to the Google maps
+            elevation API to retrieve the height of the input address.
+
+        Returns
+        -------
+        location : This class (a `~astropy.coordinates.EarthLocation` or subclass)
+            The location of the input address.
+
+        See Also
+        --------
+        https://developers.google.com/maps/documentation/geocoding/intro
```

Comment:
These lines cause (I think) the sphinx build to fail. Maybe listing them as references will help? I.e.,
```
References
----------
.. [1] https://developers.google.com/maps/documentation/geocoding/intro
.. [2] https://developers.google.com/maps/documentation/elevation/intro
```